config/sys.config.src

```diff
@@ -45,7 +45,10 @@
             {downlink_endpoint, <<"${ROUTER_CONSOLE_DOWNLINK_ENDPOINT}">>},
             {secret, <<"${ROUTER_CONSOLE_SECRET}">>}
         ]},
-        {router_xor_filter_worker, "${ROUTER_XOR_FILTER_WORKER}"},
+        {ics, #{
+            host => "${ROUTER_ICS_HOST}"
+            port => ${ROUTER_ICS_PORT:-8080}
```

Comment:
Does this syntax work for you? I can never get rebar3 to not complain about it when I launch a shell.